src/Core/Silk.NET.Core/Native/PInvokeOverride.cs

```diff
@@ -0,0 +1,17 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+
+namespace Silk.NET.Core.Native
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class PInvokeOverride : Attribute
```

Comment:
You have a choice: document this now or later because between preview 5 and 2.0 we'll be having a massive documentation sprint.

Reply:
I'll do it when I document SilkTouch + Core, don't really have the time to think about how to explain all this best.